build.gradle

```diff
@@ -35,6 +35,7 @@ buildscript {
         projectReactorVersion = '3.2.6.RELEASE'
         retrofitVersion = '2.5.0'
         rxJavaVersion = '2.2.7'
+        mockkVersion = '1.9.3'
```

Comment:
I don't believe we need to do mock testing, we use property-based testing everywhere with real data.